src/constants/index.ts

```diff
@@ -29,6 +33,10 @@ export const MCELL_URL = "https://mcell.org/";
 export const SMOLDYN_URL = "http://www.smoldyn.org/";
 export const SPRINGSALAD_URL = "https://vcell.org/ssalad";
 export const MEDYAN_URL = "http://medyan.org/";
+export const UI_TEMPLATE_URL_ROOT =
+    "https://api.github.com/repos/simularium/simulariumio/contents/ui-templates";
+export const UI_TEMPLATE_DOWNLOAD_URL_ROOT =
+    "https://raw.githubusercontent.com/simularium/simulariumio/main/ui-templates";
```

Comment:
is github considered the production location for this data?  that seems slightly sketchy. I wonder if we could pre-fetch them as part of the build process here so they live with the app?  It's ok if this is out of scope for this PR but worth thinking about.

Reply:
Definitely something I could look into, I'll leave those changes for a different PR but follow up.

Reply:
yeah, that's been the plan, so they can stay in sync with the file format on simulariumio. I do like the idea of fetching them in the build process though